guides/_subsections/navigate-the-dashboard.md

```diff
@@ -1,5 +1,5 @@
 ## [Navigate the Dashboard](#navigate-the-dashboard)
-The *Dashboard* is the first view you will see when you log into Cryostat. It provides a high-level overview of the state of your Cryostat instance and the target JVM applications it is monitoring.
+The *Dashboard* is the first content view you will see when you log into Cryostat. It provides a high-level overview of the state of your Cryostat instance and the target JVM applications it is monitoring.
```

Comment:
Cryostat > **Cryostat**

Reply:
This document has a lot more editing changes like this that would be needed, I think. It's quite a large document and not part of the new content I'm adding here, I just wanted to update this one line since it isn't quite content-accurate anymore.